client/sysinfo/src/sysinfo.rs

```diff
@@ -319,6 +334,41 @@ pub fn benchmark_disk_random_writes(directory: &Path) -> Result<u64, String> {
 
 	// We only wrote half of the bytes hence `SIZE / 2`.
 	benchmark("disk random write score", SIZE / 2, MAX_ITERATIONS, MAX_DURATION, run)
+		.map(|s| s as u64)
+}
+
+pub fn benchmark_sr25519_verify(limit: ExecutionLimit) -> f64 {
```

Comment:
nit: missing doc on public function. Does it need to be public?

Reply:
Will add :smile:   
Currently it needs to be public since I call these functions directly. IMHO it would be better to have a `gather` function which calls all of them and returns a struct,  
but thats probably next.

Reply:
> IMHO it would be better to have a `gather` function which calls all of them and returns a struct,

Hmm... although the downside of that is that you won't be able to configure them separately (set a different `ExecutionLimit` for each benchmark), unless you add another struct which will have multiple `ExecutionLimit`s in it, but then if you're going to do that you might as well have them as a separate functions I guess?

(And it might make sense to set different limits on each benchmark because e.g. the disk benchmarks are going to require more time than the memory benchmark to make sure their result isn't noisy, if you absolutely want to minimize their error.)